<?xml version="1.0" ?>
<launch>

  <node name="apply_image_mask_cam0" pkg="challenge_tools_ros" exec="apply_image_mask.py" output="screen">
    <param name="mask_filename" value="$(find-pkg-share challenge_tools_ros)/masks/cam0_mask.png"/>
    <param name="cam_topic" value="/cam0/image_raw"/>
  </node>

  <node name="apply_image_mask_cam1" pkg="challenge_tools_ros" exec="apply_image_mask.py" output="screen">
    <param name="mask_filename" value="$(find-pkg-share challenge_tools_ros)/masks/cam1_mask.png"/>
    <param name="cam_topic" value="/cam1/image_raw"/>
  </node>


</launch>
